src/internal/widget-exports.ts

```diff
@@ -0,0 +1,19 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+export { PACKAGE_VERSION } from './environment';
+export {
```

Comment:
Do lines 4 to 14 introduce duplication with `src/app-layout/visual-refresh-toolbar/exports.tsx`? Is the plan to remove one of the two?

Reply:
Yes, the plan is to remove the old file